src/malco/post_process/compute_mrr.py

```diff
@@ -4,15 +4,29 @@
 import pandas as pd
 import pickle as pkl
 from malco.post_process.mondo_score_utils import score_grounded_result
+from malco.post_process.mondo_score_utils import omim_mappings
+from typing import List
+from oaklib.interfaces import OboGraphInterface, MappingProviderInterface
 
 
+from oaklib import get_adapter
+
+
+def mondo_adapter() -> OboGraphInterface:
+    """
+    Get the adapter for the MONDO ontology.
+
+    Returns:
+        Adapter: The adapter.
+    """
+    return get_adapter("sqlite:obo:mondo") # , cache_lookups=True
```

Comment:
why would not cache_loopups make this faster?

Reply:
Another comment that I forgot to delete. cache_lookups is a oaklib.interfaces.MappingProviderInterface specific function. It makes the code significantly slower for some reason. We are caching anyway with cachetools. If you are wondering, why not use memoize? This is exactly it. Why not use functools.lrucache as before? Because then we cannot, to the best of my extended research, pass the mondo adapter as an argument and actually need to initialize multiple times

Reply:
Moved to 

https://github.com/monarch-initiative/malco/issues/30